codepen/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "author": {
+      "name": "Chezzer",
+      "id": "195979856733929472"
+    },
+    "url": [
+        "codepen.io",
+        "www.codepen.io"
+      ],
+    "description": {
+      "en": "CodePen is an online community for testing and showcasing user-created HTML, CSS and JavaScript code snippets."
+    },
+    "service": "CodePen",
+    "version": "1",

```

Comment:
This isn't required, but I think you should set this to `1.0.0` instead of `1`. It's more standard.